bayesian/api_v1.py

```diff
@@ -22,6 +22,7 @@
                               get_component_percentile_rank, usage_rank2str,
                               MavenCoordinates, case_sensitivity_transform)
 from f8a_worker.manifests import get_manifest_descriptor_by_filename
+from f8a_worker.setup_celery import init_selinon
```

Comment:
I think this shouldn't be needed any more. We initialize selinon only once in https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/__init__.py#L85.